src/Cli/Microsoft.TemplateEngine.Cli/Commands/SharedOptions.cs

```diff
@@ -11,7 +12,23 @@ public static class SharedOptions
         {
             Description = SymbolStrings.Option_Output,
             Required = false,
-            Arity = new ArgumentArity(1, 1)
+            Arity = new ArgumentArity(1, 1),
+            CustomParser = (ArgumentResult argumentResult) =>
+            {
+                var input = argumentResult.Tokens[0]?.Value;
```

Comment:
if the argument is null/empty, shouldn't the parser raise an error or is that checked before hitting the customer parser?